### ISSUE TITLE
The "quickfix: Use type signature:" action doesn't honour qualified imports

### ISSUE THREAD

**Issue by googleson78** [haskell/haskell-language-server#441]
### Subject of the issue
If I have `import qualified Data.Text as T` and I try to autoinsert a type signature for something with a `Text` argument, only `Text` will be inserted, as opposed to `T.Text`, even though there is no `Text` in scope.

### Your environment
HLS version:
```
haskell-language-server version: 0.4.0.0 (GHC: 8.8.4) (PATH: /home/googleson78/.local/bin/haskell-language-server) (GIT hash: 066ce8b94f3be66a011818f748ecb6c6ab7974b8)
```

I can paste all the other info if it turns out to be necessary, but this seems like a "logic bug" as opposed to an environment/setup one.

**Comment by berberman** [comment]
It seems that the `FileDiagnostic` returned by the type check in ghcide doesn't contain qualifiers, thus they don't present in warning messages, and `suggestSignature` can't grab them...

**Comment by pepeiborra** [comment]
Closed by haskell/ghcide#938
